src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +271,52 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool IsAccessorListWithoutAccessorsWithBlockBody(SyntaxNode? node)
+            => node is AccessorListSyntax accessorList &&
+                accessorList.Accessors.All(a => a.Body == null);
+
+        private static bool HasInitializer(BasePropertyDeclarationSyntax basePropertyDeclaration)
+            => basePropertyDeclaration is PropertyDeclarationSyntax property && property.Initializer != null;
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxNode? node)
```

Comment:
`IsAccessorListOfPropertyWithInitializer`

Reply:
@CyrusNajmabadi Both the old method names are fine for @AlekseyTs 
Please tell me if you both want to stay with the ones that are currently committed of should I rename them :)